src/pages/graphs-visualizations.html

```diff
@@ -96,7 +96,8 @@ <h3>{{configLoaded.name}}</h3>
                                 text-button=""
                                 visual-button="Show"
                                 visual-callback="openUri(uri)"
-                                empty="empty">
+                                empty="empty"
+                                starting-i-r-i-example-value="{{configLoaded.example}}">
```

Comment:
Actually, my suggestion was to put the configLoaded.example value in the paragraph above:
<p>Search for an IRI to view it. <span ng-if="configLoaded.name != defaultGraphConfig.name">This is a user defined advanced configuration.</span>Expected IRI: {{configLoaded.example}}</p> As this is the text that defines what to type in the input. The autocomplete hint is rather related to how to autocomplete, not what value the IRI should be. So better remove it from there. Sorry for not being clear enough previous time.